lib/rollup/data/totals_data.ex

```diff
@@ -1,53 +1,35 @@
 defmodule Castle.Rollup.Data.Totals do
   import Castle.Rollup.Jobs.Totals
+  import Castle.Redis.HashCache
+  alias Castle.Redis.Interval.Getter, as: Getter
 
-  # TODO: this is real ugly. Just trying to combine/sort the episode maps
-  def podcasts() do
-    {result, _meta} = get()
-    result
-    |> Enum.map(&(Map.put(&1, :feeder_episodes, [&1.feeder_episode])))
-    |> Enum.map(&(Map.delete(&1, :feeder_episode)))
-    |> Enum.group_by(&(&1.feeder_podcast))
-    |> Enum.map(fn({_podcast_id, episodes}) ->
-      Enum.reduce(episodes, %{}, fn(episode, acc) ->
-        Map.merge(acc, episode, fn(k, v1, v2) ->
-          case k do
-            :count -> v1 + v2
-            :feeder_episodes -> Enum.sort(v1 ++ v2)
-            _ -> v2
-          end
-        end)
-      end)
-    end)
-    |> Enum.sort(&(&1.feeder_podcast < &2.feeder_podcast))
-  end
+  # TODO: this is a bit inside-baseball
+  @podcast_downloads_key "downloads.podcasts.DAY"
+  @episode_downloads_key "downloads.episodes.DAY"
+  @daily BigQuery.TimestampRollups.Daily
 
   def podcast(id) do
-    podcasts() |> Enum.find(&(&1.feeder_podcast == id))
+    hash_fetch podcasts_key(), id, fn(from) ->
```

Comment:
So this gets all data "before today" from the `rollups.totals.podcasts` redis hash.  And then pulls the "live" data for today from `downloads.episodes.DAY.2018-02-16T00:00:00Z`.